PubFactory Journals.js

```diff
@@ -0,0 +1,716 @@
+{
+	"translatorID": "8d1fb775-df6d-4069-8830-1dfe8e8387dd",
+	"label": "PubFactory Journals",
+	"creator": "Brendan O'Connell",
+	"target": "^https://([^/]+/view/journals/.+\\.xml|[^.]*journals\\.[^/]+/search)\\b",
+	"minVersion": "5.0",
+	"maxVersion": "",
+	"priority": 200,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2023-08-29 11:01:36"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2023 Brendan O'Connell
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc) {
+	// EM works well on single articles, so detectWeb() isn't strictly necessary for them.
+	// However, single articles are included in this translator
+	// so that 'multiple' doesn't get called for articles with "Related Items"
+	// e.g. https://avmajournals.avma.org/view/journals/javma/261/4/javma.22.11.0518.xml
+	// because there are multiple items that match the rows selector in getSearchResults and
+	// haven't found a way to write a querySelectorAll() that only matches the main article on single pages
+	// with related items.
+	if (doc.querySelector('meta[property="og:type"][content="article"]')) {
```

Comment:
The problem is that this would return true on both single article pages (e.g. https://journals.humankinetics.com/view/journals/ijsnem/33/2/article-p73.xml) and `multiple` (e.g. https://journals.humankinetics.com/view/journals/jab/jab-overview.xml), and I need it in `detectWeb()` to differentiate between the two. I agree with you that the selector is very broad, but the regex target for the translator isn't: `"^https://([^/]+/view/journals/.+\\.xml|[^.]*journals\\.[^/]+/search)\\b",`, I haven't seen a lot of other journals that use .xml file extensions for their pages. If this is still problematic, though, I can definitely look for a better selector for individual article pages.